04_Running_The_Class.Rmd

```diff
@@ -0,0 +1,285 @@
+
+At the time of writing (Feb 2023) Terra is working to expedite this process for Terra users; we recommend checking the [relevant Terra documentation](https://support.terra.bio/hc/en-us/articles/6101030164507-When-workspace-creation-fails-Google-project-quotas) for the latest information as well as recommendations about how to fill out the form.
+
+In our experience, the process has been reasonably straightforward and quick (approved in a couple of days).
+
+
+# Set Up Student Accounts
+
+In order for your students to compute on AnVIL, they will need to set up an account and provide you with their username so that you can grant them access to resources (Billing Projects and Workspaces).
+
+If possible, it can be helpful to do this ahead of time, so that you do not need to dedicate class time for it.
+
+<br>
+
+::: {.warning}
+Make sure you understand when and where your students can spend money!
+
+The following actions will give your students the ability to spend money on AnVIL:
+
+- Adding them to a **Billing Project** - this will let them create their own Workspaces
+- Adding them as **Writers or Owners to a Workspace** - this will let them compute in the Workspace
+:::
+
+Our recommendation is to **do most set up ahead of time**:
+
+- Create Student Group (and add Instructors)
+- Have students create accounts (and collect usernames)
+- Add students to the Student Group
+- Give the Student Group access to any Workspaces **as Readers**.  This will let them view the contents but not spend money.
+
+**Wait until shortly before the students need to compute** to add the Student Group to the Billing Project.  This prevents them from starting something up early and then accruing costs by failing to shut it down.
+
+This chapter walks through all of the student setup steps that can safely be done ahead of time.  The final step for activating student billing is covered in the next chapter.
+
+
+## Create Student Group {#student-group}
+
+Creating a Group for your students will let you easily manage the class as a whole by granting the Group access to Billing Projects and Workspaces.
+
+You will need a unique name for your Group. We suggest a combination of institution-class-role-term (e.g., `jhu-bio101-students-2023FA`). Only letters, numbers, underscores, and dashes are allowed in Group names.
+
+To create a Group:
+
+::::{.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_create.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+You now have a unique **Student Group**.
+
+### Add Instructors to Group
```

Comment:
I think this section is kind of redundant with instructor-group.html#add-instructors-and-tas-to-group - can we combine?